src/adapters/suggest.js

```diff
@@ -154,9 +107,6 @@ export default class Suggest {
 
   async onSubmit() {
     if (this.pending) {
-      if (this.bragiPromise) {
-        this.bragiPromise.abort();
-      }
```

Comment:
This change has actually a small side-effect:
without this abort, the suggestions results may be briefly re-rendered after the submit action occurred (and the pending request returned its results).

![Peek 19-03-2020 11-10](https://user-images.githubusercontent.com/4726554/77056684-120bca80-69d3-11ea-98ff-491c1cd6d8ff.gif)

Reply:
Arf, nevermind. This behavior is also present on "master".